### DISCUSSION TITLE
Suggestions while typing title

### DISCUSSION THREAD

**Discussion by totti4ever** [jonaswinkler/paperless-ng#557] [Feature Requests]
Hey, I have quite a few recurring documents, which have the same or a similar title like `paycheck 2021-02` or `tax receipt 2020`. It would help a lot if while typing suggestions from already existing document titles would appear to choose from.

Of course, this might distract some people - but for me it would help a lot to meet a common naming pattern of those documents!

**Comment by BlankSlate56** [comment]
I've more or less given up on titles.  I have my file naming rule set up so it renames the files correspondent/document type/title/created date.  I then only use the Title field for specific identifying data (check numbers, invoice numbers, etc) when it is present.  So, I'll end up with a file named BestBuy Invoice 9999 12-1-2020.  If there isn't an invoice number and I leave the title blank, then it outputs BestBuy Invoice  12-1-2020 into the filename.  This isn't ideal as I don't have useful titles to work with in Paperless, but I'm not going to go through manually typing a title for every single document I scan in when the whole point of the machine learning for me is to pull that data out automatically.  I'd really like to be able to create a Title rule similar to how I can create the file naming rules.  It is working for how I use the app but I wish it were better.  It is still far better than manually renaming files with file explorer.

**Reply by totti4ever** [reply]
I think I understand what you mean. For me the main aspect to do without titles would be a free text search where the results could at least be ordered by date.

But have I should give the auto tagging a bigger role, though nested tags would help with that ✌️

**Reply by BlankSlate56** [reply]
I manually enter many of my "Created" dates because it seems that the machine learning struggles with pulling the dates for some reason and it is a known issue.  I didn't realize until now that doing a free text search doesn't give you the ability to sort in the same way you can from the regular documents view.  Still, I mainly search for docs using the drop-filters from the documents view and not so much with the word search.
